homeassistant/components/uptimerobot/quality_scale.yaml

```diff
@@ -0,0 +1,80 @@
+rules:
```

Comment:
Why is `monitor_available` a field on the entity.py? Can't that just be in the binary_sensor

Reply:
Will move it in a separate PR

Reply:
https://github.com/home-assistant/core/pull/142986